README.rst

```diff
@@ -89,7 +89,7 @@ Requirements
 ------------
 
 1. Python (3.6, 3.7, 3.8, 3.9)
-2. Django (2.2, 3.0, 3.1, 3.2)
+2. Django (2.2, 3.0, 3.1, 3.2, 4.0)
```

Comment:
django 3.0 is EOL already so can be dropped from CI/Docs

Reply:
We will remove this in the next DJA major version 5.0.0 as such a change breaks backwards compatible. As in current master we have new deprecation warnings the next DJA version will be 4.3.0 though (in major releases we do not want to introduce new deprecation).

On a side note, I appreciate that you keep an eye on dependencies being EOL. Instead of commenting in an unrelated PR (this PR is about adding Django 4.0 support) I would prefer that you create an issue like "Remove support for EOL Django 3.0". This can then be used as an action item where as a comment in an unrelated PR might just be overlooked and forgotten about. Thanks for understanding.

Reply:
ok